cpp/ql/src/Best Practices/Magic Constants/JapaneseEraDate.qhelp

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+  <p>
+    When eras change, date and time conversions that rely on a hard-coded era start date need to be reviewed. Conversions relying on Japanese dates in the current era can produce an ambiguous date. 
+    The values for the current Japanese era dates should be read from a source that will be updated, such as the Windows registry.
```

Comment:
We should add some guidance on handling this on Linux and MacOS systems as well.

Reply:
Do you know what the right guidance might be?